### PR TITLE
Feature: add property `isMirrored` in MTHKView and GLHKView

### PR DESCRIPTION
**Pull request by allenlinli**
# Description
I add `isMirrored` property for GLHKView and MTHKView, so the view can be mirrored when `rtmpStream.captureSettings[.isVideoMirrored]` is set true.

I did not add a mirror button on the UI, because there's no button for setting `rtmpStream.captureSettings[.isVideoMirrored]` to true yet. We may need manual testing for this part, or I can add the UI later if needed.

I hope this new property is clear and useful enough.

### CODE REVIEW

**Comment by allenlinli** [comment]
This PR is the same as https://github.com/shogo4405/HaishinKit.swift/pull/701 
That PR was closed by mistake.

Furthermore, I made the properties open.
I am not sure either `open` or `public` is more suitable for this property.

File: Platforms/iOS/MTHKView.swift [hunk 1]
```diff
@@ -103,9 +104,18 @@ extension MTHKView: MTKViewDelegate {
...
             .transformed(by: CGAffineTransform(scaleX: scaleX, y: scaleY))
+
+        if isMirrored {
+            if #available(iOS 11.0, *) {
+                scaledImage = scaledImage.oriented(.upMirrored)
+            } else {
+                scaledImage = scaledImage.oriented(forExifOrientation: 2)
+            }
+        }
+
```
Comment (hambly) [inline Platforms/iOS/MTHKView.swift:118]: The orientation needs to happen before the transformation. @shogo4405

Comment (allenlinli) [reply Platforms/iOS/MTHKView.swift:118]: Hi @hambly, I am not familiar with the CGAffineTransform part, can you explain more about why does the orientation needs to happen before the transformation?
The appended PR and comments are welcome, too.

Comment (hambly) [reply Platforms/iOS/MTHKView.swift:118]: Orientation is inherent to the media, the transformation is part of rendering for display, in particular the translation aspect.

Comment (allenlinli) [reply Platforms/iOS/MTHKView.swift:118]: So you mean the results would be the same here, but we should usually do orientation before transformation for the process, correct?

Comment (hambly) [reply Platforms/iOS/MTHKView.swift:118]: The results are not the same here, this PR did not correctly configure the mirrored front-facing camera, as the translation becomes incorrect upon mirroring.

See https://github.com/shogo4405/HaishinKit.swift/pull/675

Comment (allenlinli) [reply Platforms/iOS/MTHKView.swift:118]: I am not sure how can the translation becomes incorrect upon mirroring...
May I ask how did the PR https://github.com/shogo4405/HaishinKit.swift/pull/675/files being closed? Is it a correct PR that should be merged?

You can propose a PR to amend this one, or I will clarify it with and give a PR based on your suggestion.

**Pull request merged**